src/e2e/specs/purchase.spec.ts

```diff
@@ -5,61 +5,39 @@
-        description:
-          "https://testrail.stage.mozaws.net/index.php?/cases/view/2463564",
-      });
-
-      // this test runs through the welcome scan flow, increasing timeout to address it
-      test.slow();
-
-      setEnvVariables(process.env.E2E_TEST_ACCOUNT_EMAIL as string);
-
-      // speed up test by ignore non necessary requests
-      await page.route(/(analytics)/, async (route) => {
-        await route.abort();
-      });
-
-      // start authentication flow
-      await landingPage.open();
-      await landingPage.goToSignIn();
-
-      // Fill out sign up form
-      const randomEmail = `_${Date.now()}@restmail.net`;
-      await authPage.signUp(randomEmail, page);
-
-      // wait for welcome page
-      await page.waitForURL("**/user/welcome");
-
-      // confirm get started step elements
-      expect(await welcomePage.getStartedStep.count()).toEqual(3);
-      await expect(page.getByText("Get started")).toBeVisible();
-      await expect(page.getByText("Enter info")).toBeVisible();
-      await expect(page.getByText("Find exposures")).toBeVisible();
-
-      // navigate to enter info step
-      await welcomePage.startFreeScanButton.click();
-
-      // confirm enter info step
-      await welcomePage.firstNameInputField.fill("Monitor");
-      await welcomePage.lastNameInputField.fill("Automation1");
-      await welcomePage.cityStateInputField.pressSequentially(
-        "Atlanta, GA, USA",
-      );
-      await page.getByText("AtlantaGA, USA", { exact: true }).click();
-      await welcomePage.dobInputField.fill("2002-01-01");
-      await welcomePage.findExposuresButton.click();
-
-      await welcomePage.modalConfirmButton.click();
-      // Waiting for scan to complete
-      await dashboardPage.actionNeededTab.waitFor();
-      expect(page.url()).toContain("/user/dashboard");
-    },
-  );
+test.describe(`${process.env.E2E_TEST_ENV} - Breach Scan, Monitor Plus Purchase Flow Yearly`, () => {
+  test.beforeEach(async ({ page, authPage, landingPage, welcomePage }) => {
+    test.info().annotations.push({
+      type: "testrail id",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463564",
+    });
+
+    test.slow(
+      true,
+      "this test runs through the welcome scan flow, increasing timeout to address it",
+    );
+
+    setEnvVariables(process.env.E2E_TEST_ACCOUNT_EMAIL as string);
+
+    // speed up test by ignoring non necessary requests
+    await page.route(/(analytics)/, async (route) => {
+      await route.abort();
+    });
+
+    // start authentication flow
+    await landingPage.open();
+    await landingPage.goToSignIn();
+
+    // Fill out sign up form
+    const randomEmail = `_${Date.now()}@restmail.net`;
+    await authPage.signUp(randomEmail, page);
+
+    // wait for welcome page
+    await page.waitForURL("**/user/welcome");
+    await welcomePage.goThroughFirstScan();
+    expect(page.url()).toContain("/user/dashboard");
```

Comment:
could also use [`.toHaveURL()`](https://playwright.dev/docs/api/class-pageassertions#page-assertions-to-have-url) with a urlOrRegExp. 🤷